### PR TITLE
Added link to home page on rooms

### PR DESCRIPTION
**Pull request by AkshatJindal1**

### CODE REVIEW

**Comment by AkshatJindal1** [comment]
Issue link: https://github.com/MakeContributions/markdown-dungeon/issues/163

@ming-tsai please have a look

**Comment by ming-tsai** [comment]
Nice work @AkshatJindal1,
But could you move this to here
image

**Comment by AkshatJindal1** [comment]
@ming-tsai implemented the requested changes. Please verify

File: src/components/layout.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default function Layout({ children }) {
   const style = {
     margin: `2rem auto`,
     maxWidth: 650,
-    padding: `2rem 2rem`,
+    padding: `5px 5px`,
```
Comment (ming-tsai) [inline src/components/layout.jsx:8]: Change this to `0.5rem`
```suggestion
    padding: `0.5rem`,
```

File: src/templates/blog-post.jsx [hunk 2]
```diff
@@ -15,8 +17,12 @@ export default function BlogPost({ data }) {
   // highlight-end
   return (
     <Layout>
-      <div dangerouslySetInnerHTML={{ __html: html }} />
-    </Layout>
+      <Link href="#" onClick={() => window.location.href = '/'} color="primary"><Home /></Link>
+      <br />
+      <div style={{ padding: `2rem 2rem` }}>
```
Comment (ming-tsai) [inline src/templates/blog-post.jsx:22]: Change this to `1.5rem` and delete `<br />`
```suggestion
      <div style={{ padding: `1.5rem 1.5rem` }}>
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Nice, some comments for styling

File: src/templates/blog-post.jsx (hunk 2 above)
Comment (AkshatJindal1) [reply src/templates/blog-post.jsx]: @ming-tsai made the changes

**Review by ming-tsai — APPROVED** [review]
Nice work @AkshatJindal1

**Pull request merged**